Java/input/application/java.xml

```diff
@@ -0,0 +1,271 @@
+<Group id="settings">
+<title>Required Java Settings</title>
+<description>To support a secured and compliant configuration, a number of 
+settings need to be modified from there default configuration and locked so 
+that they are prevented from being changed.</description>
+
+<Rule id="java_jre_deployment_config_exists" severity="medium">
+<title>The Java Configuration File Exists</title>
+<description>
+If the Java configuration file does not exist, it can be added by running:
+<pre>$ sudo mkdir -p -m 755 /etc/.java/deployment
+$ sudo touch /etc/.java/deployment/deployment.config 
+$ sudo chmod 644 /etc/.java/deployment/deployment.config</pre>
+</description>
+<ocil clause="it does not exist">
+To verify that the <tt>deployment.config</tt> file exists,
+run the following command:
+<pre>$ ls /etc/.java/deployment</pre>
+The output should contain:
+<pre>deployment.config</pre>
+</ocil>
+<rationale>
+The deployment.config file is used for specifying the location and execution of
+system-level properties for the Java Runtime Environment. By default no
+deployment.config file exists; thus, no system-wide deployment.properties file exists.
+Without the deployment.config file, setting particular options for the Java
+control panel is impossible.
+</rationale>
+<oval id="java_jre_deployment_config_exists" />
+<ident cce="" stig="JRE0070-UX" />
+<ref nist="DCBP-1" disa="" />
+</Rule>
+
+<Rule id="java_jre_deployment_config_configured" severity="medium">
+<title>The Java Configuration File Is Configured Correctly</title>
+<description>
+To ensure that the Java <tt>/etc/.java/deployment/deployment.config</tt> file is
+configured correctly, add or modify
+<tt>deployment.system.config</tt> to equal 
+<tt>file:///etc/.java/deployment/deployment.properties</tt>. Also add or modify
+<tt>deployment.system.config.mandatory</tt> to equal <tt>false</tt>.
```

Comment:
Should this be broken into two XCCDF rules, since two configuration actions are required?